examples/python3/useful/header.py

```diff
@@ -78,20 +78,26 @@ def addHeader(parentNode, multithreading=False, inverse=False, isConstrained=Fal
                                  multithreading=multithreading, epsilon=1)
         else:
             parentNode.addObject('GenericConstraintSolver', name='ConstraintSolver', tolerance=1e-8, maxIterations=100,
-                                 multithreading=multithreading)
+                                 multithreading=multithreading, printLog=1)
 
     if isContact:
-        contactHeader(parentNode)
+        contactHeader(parentNode, _contact_params=params.contactParams)
 
 
 # components needed for contact modeling
-def contactHeader(parentNode):
+def contactHeader(parentNode, _contact_params=None):
```

Comment:
There is a general lack of consistency in the code base regarding the naming convention.
Is it CAML case like "parentNode" or PEP8 style like "parent_node". 

Whatever the convention is, please stick to one and only one. 
In addition why "_contact_params" has a leading  underscore ?